dockerfiles/agent/cron/backup-cron-job

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2019-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+
+# run backup.sh as cron task every 10 minutes
+*/10 * * * * /scripts/backup.sh
```

Comment:
in the future would be nice to somehow configure the backup time ()

Reply:
won't env vars work here? if no - could we copy into container template that will be initialized with env vars values by entrypoint script?

Reply:
Created issue for this
https://github.com/che-incubator/workspace-data-sync/issues/3